packages/blocks/src/page-block/edgeless/components/toolbar/brush/brush-tool-button.ts

```diff
@@ -75,6 +91,14 @@ export class EdgelessBrushToolButton extends LitElement {
     }
   }
 
+  private _handleMouseEnter() {
+    this._penIcon.setAttribute('viewBox', '0 0 22 60');
+  }
+
+  private _handleMouseLeave() {
+    this._penIcon.setAttribute('viewBox', '0 0 22 52');
+  }
```

Comment:
This is not the best practice. Can we use `transform` to adjust its size?

Reply:
I have tried to use :hover selector and transform: translateY(-8px) (or transform: scale()), but it seems like cannot achieve the expected effect. The expected effect is when hover on the pen , the pen will stretch by 8px. Is there a better way to achieve this?

Reply:
Have implemented by css